pkgs/tools/typesetting/tex/texlive/default.nix

```diff
@@ -67,12 +67,13 @@ let
     # used for non-final snapshots and as fallback for final snapshots that have not reached yet the historic mirrors
     # please note that this server is not meant for large scale deployment and should be avoided on release branches
     # https://tug.org/pipermail/tex-live/2019-November/044456.html
+    "https://mirror.ox.ac.uk/sites/ctan.org/systems/texlive/tlnet"
```

Comment:
Oops, that won't build starting from today (CTAN is surely distributing a new `tlpdb.xz` already!).